### DISCUSSION TITLE
Release zip file 1.12.2 contains a single zip file

### DISCUSSION THREAD

**Discussion by cweiske** [Kovah/LinkAce#677] [Ideas & Feature Requests]
The release zip file 1.12.2 `linkace-v1.12.2.zip` only contains a zip file `linkace.zip`, which in turn contains the application code.

It would be nice if I would not need to unzip two files to extract the application.

https://github.com/Kovah/LinkAce/releases/download/v1.12.2/linkace-v1.12.2.zip

**Comment by Kovah** [comment]
Thanks for the heads up, will do with the next release.
